BootloaderCorePkg/Tools/CommonUtility.py

```diff
@@ -34,6 +34,38 @@ class LZ_HEADER(Structure):
         'LZMA' : 'Lzma',
     }
 
+class RsaKey (Structure):
+    _fields_ = [
+        ('PubExp',  ARRAY(c_uint8, 4)),
+        ('Modulus', ARRAY(c_uint8, 0)),      #Size for RSA2K/RSA3K in bytes
+        ]
+
+class PublicKey (Structure):
+    _pack_ = 1
+    _fields_ = [
+        ('Identifier', ARRAY(c_char, 4)),      #signature ('P', 'U', 'B', 'K')
+        ('KeySize',    c_uint16),              #Length of Public Key
+        ('KeyType',    c_uint8),               #RSA or ECC
+        ('Reserved',   ARRAY(c_uint8, 1)),
+        ('KeyData',    ARRAY(c_uint8, 0)),   #Pubic key data with KeySize bytes for RsaKey() format
+        ]
+
+    def __init__(self):
+        self.Identifier = b'PUBK'
+
+class Signature (Structure):

```

Comment:
Let us use the same name as C code:  SIGNATURE_HDR

Reply:
Yes, will do.